From 040ff334cf79a2f66374ee415360a3fcaf3534dc Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyun.kwon@xilinx.com>
Date: Tue, 11 Apr 2017 09:03:47 -0700
Subject: [PATCH 0207/1851] media: xilinx: tpg: Block comments

commit 512862d0cdc840c306bb7175bf9e76e178435206 from
https://github.com/Xilinx/linux-xlnx.git

Block comments by using */ on a seprate line:

./scripts/checkpatch.pl --max-line-length 120 xilinx-tpg.c
WARNING: Block comments use a trailing */ on a separate line
+	 * the sink pad size */

total: 0 errors, 1 warnings, 1168 lines checked

Signed-off-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-tpg.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/media/platform/xilinx/xilinx-tpg.c b/drivers/media/platform/xilinx/xilinx-tpg.c
index 498cd0c396f9..059629281853 100644
--- a/drivers/media/platform/xilinx/xilinx-tpg.c
+++ b/drivers/media/platform/xilinx/xilinx-tpg.c
@@ -409,7 +409,8 @@ static int xtpg_enum_frame_size(struct v4l2_subdev *subdev,
 
 	/* Min / max values for pad 0 is always fixed in both one and two pads
 	 * modes. In two pads mode, the source pad(= 1) size is identical to
-	 * the sink pad size */
+	 * the sink pad size.
+	 */
 	if (fse->pad == 0) {
 		fse->min_width = XVIP_MIN_WIDTH;
 		fse->max_width = XVIP_MAX_WIDTH;
-- 
2.31.1

